src/commands/runtime/activation/logs.js

```diff
@@ -59,7 +59,7 @@ class ActivationLogs extends RuntimeBaseCommand {
         filterActions.push(flags.action)
       }
 
-      rtLib.printActionLogs({ ow: owOptions }, this.log, limit, filterActions, flags.strip, flags.poll || flags.tail || flags.watch)
+      await rtLib.printActionLogs({ ow: owOptions }, this.log, limit, filterActions, flags.strip, flags.poll || flags.tail || flags.watch)
```

Comment:
LGTM. Could the issue be addressed by `returning`? I'm asking because it's clear where the responsibility holds for handling errors from `printActionLogs` - it looks like it will be some external caller, in which case I see no difference between await and return.

Reply:
This should be upstreamed, the AIO tests didn't complain about this so they need to be tightened. We should open an issue upstream for it. Happy to do that or feel free to Josh if you prefer.